File: Code/Websites/DanpheEMR/Services/Verification/DTOs/Pharmacy/PharmacySubStoreRequisitionItemVerification_DTO.cs
using System;

namespace DanpheEMR.Services.Verification.DTOs.Pharmacy
{
    public class PharmacySubStoreRequisitionItemVerification_DTO
    {
        public int RequisitionItemId { get; set; }
        public int ItemId { get; set; }
        public double Quantity { get; set; }
        public double PendingQuantity { get; set; }
        public int RequisitionId { get; set; }
        public string RequisitionItemStatus { get; set; }
        public string Remark { get; set; }
        public double CancelQuantity { get; set; }
        public int? CancelledBy { get; set; }
        public DateTime? CancelledOn { get; set; }
        public string CancelRemarks { get; set; }
        public string Unit { get; set; }
        public virtual PharmacySubStoreRequisitionVerification_DTO Requisition { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/verification/pharmacy/shared/pharmacy-substore-requisition-item-verification.dto.ts
export class PharmacySubStoreRequisitionItemVerification_DTO {
    public RequisitionItemId: number;
    public ItemId: number;
    public ItemName: string;
    public Quantity: number;
    public PendingQuantity: number;
    public RequisitionId: number;
    public RequisitionItemStatus: string;
    public Remark: string;
    public CancelQuantity: number;
    public CancelledBy: number;
    public CancelledOn: string;
    public CancelRemarks: string;
    public Unit: string;
    public IsActive: boolean = false;
    public IsEdited: boolean = false;
}
